plane/src/drone/runtime/docker/mod.rs

```diff
@@ -230,10 +230,10 @@ impl Runtime for DockerRuntime {
                 "Container not found, assuming it was already terminated."
             );
         } else {
-            return result.map_err(|e| e.into());
+            return Ok(false);
```

Comment:
From offline convo: this should stay the same and the `return Ok(false)` should move into the prev block